server/src/main/java/org/apache/iotdb/db/auth/AuthorityChecker.java

```diff
@@ -315,6 +315,10 @@ private static int translateToPermissionId(StatementType type) {
       case FILL:
       case GROUP_BY_FILL:
       case SELECT_INTO:
+      case SHOW_SCHEMA_TEMPLATE:
```

Comment:
Since `READ_TIMESERIES` is a path-relevant entry, I am not sure what it is supposed to show if a user issued a `SHOW_SCHEMA_TEMPLATE` with limited privilege on some paths?

Reply:
Fixed.